test_runner/regress/test_sharding.py

```diff
@@ -261,7 +262,9 @@ def test_sharding_split_compaction(neon_env_builder: NeonEnvBuilder, failpoint:
     env.pageserver.start()
 
     # Cleanup part 2: once layers are outside the PITR window, they will be rewritten if they are partially redundant
-    env.storage_controller.pageserver_api().set_tenant_config(tenant_id, {"pitr_interval": "0s"})
+    updated_conf = TENANT_CONF.copy()
```

Comment:
Should we use `copy.deepcopy(...)` here? (I'm not sure if `TENANT_CONF` can include other structures like dicts/list, but just in case)

Reply:
It could in principle, but not at present -- if we were making a more generic helper I'd probably deepcopy, but for here let's use the minimum we need.